lib/src/widgets/post/post_something.dart

```diff
@@ -25,17 +25,13 @@ class PostSomething extends StatelessWidget {
           : () => toast("You do not have permission to create a post"),
       child: Container(
         width: screenSize.width,
-        height: 92,
-        color: Colors.transparent,
-        padding: const EdgeInsets.symmetric(horizontal: 20.0, vertical: 16.0),
+        height: 60,
```

Comment:
Why did we decrease height, and add padding?

Reply:
It was taking extra space which was the issue reported by suraasa
- Removed vertical padding which in turn decreased the overall height of the widget